modules/cypress/vue/providers/CypressProvider.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <slot />
+</template>
+
+<script setup lang="ts">
+import { onMounted } from 'vue'
+
+onMounted(() => {
+  setTimeout(
+    () =>
+      // @ts-ignore
+      (window.sfx = true),
```

Comment:
Maybe rename it to something more descriptive and something that has lower chance of colliding. Maybe window._sfxCypressMounted ?